src/flint/nmod.jl

```diff
@@ -54,6 +54,13 @@ isone(a::zzModRingElem) = (a.parent.n == 1) || (a.data == 1)
 
 modulus(R::zzModRing) = R.n
 
+function krull_dim(R::zzModRing)
+  @req !is_trivial(R) "`krull_dim` is not supported for trivial rings"
+  return is_prime(modulus(R)) ? 0 : 1
```

Comment:
NO! If the modulus is prime, a longest chain starts at the modulus otherwise it starts at an associated prime of the modulus and <0> does not appear in the chain, as it is not prime. In both cases the length of the chain is zero!